src/_pytest/reports.py

```diff
@@ -82,9 +82,10 @@ def toterminal(self, out: TerminalWriter) -> None:
             longrepr.toterminal(out)
         else:
             try:
-                out.line(longrepr)
```

Comment:
@bluetech do you know why `mypy` didn't catch this problem? `TerminalWriter.line` expects a `str` argument, but `longrepr` is declared as `Any` in the class definition.

Reply:
a function which accepts `str` also accepts `Any` because it's gradually typed (there's options to turn this off in mypy, but getting to that ideal is very very difficult)

Reply:
Ahh I indeed:

[Any](https://docs.python.org/3/library/typing.html#the-any-type)

> A static type checker will treat every type as being compatible with Any and Any as being compatible with every type.

Thanks! My thought was that a function declaring `str` would accuse `Any` as incompatible, but the above definition clearly states otherwise. 👍